src/lib/operators/abstract_operator.cpp

```diff
@@ -9,7 +12,11 @@ AbstractOperator::AbstractOperator(const std::shared_ptr<const AbstractOperator>
 void AbstractOperator::execute() { _output = _on_execute(); }
 
 std::shared_ptr<const Table> AbstractOperator::get_output() const {
-  // TODO(student): You should place some meaningful checks here
+  // ensure we return an empty chunk only if the chunk count is also one
+  if (_output->get_chunk(ChunkID{_output->chunk_count() - ChunkID{1}})->size() == 0) {
+    Assert(_output->chunk_count() == ChunkID{1},
+           "Detected an empty chunk in an operator result with 2 or more chunks.");
+  }
```

Comment:
I like this check! 
I believe that the assertion is only evaluated if the last chunk is empty. What about potential empty chunks in the middle of the table?